drivers/hsc/include/hsc_internals.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2020 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_hsc
+ * @brief       Internal addresses, constants for the HSC sensor.
+ * @{
+ *
+ * @file
+ * @brief       Internal addresses, constants for the HSC sensor.
+ *
+ * @author      Quang Pham <phhr_quang@live.com>
+ */
+
+#ifndef HSC_INTERNALS_H
+#define HSC_INTERNALS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    HSC I2C Address
+ * @{
+ */
+#define HSC_ADDR                    (0x28)   /* 7 bit address */
+/** @} */
+
+/**
+ * @name    HSC Max Range
+ * @{
+ */
+#define HSC_RANGE                   (40U)    /* in mbar dimension */
+/** @} */
+
+/**
+ * @name    HSC I2C Packet Readout
+ * @{
+ */
+#define HSC_PRESSURE_DATA_LENGTH    (2)      /* Pressure is stored in the first 2 bytes of data */
+#define HSC_FULL_DATA_LENGTH        (4)	     /* Pressure + temperature data is 4 bytes long */
```

Comment:
```suggestion
#define HSC_FULL_DATA_LENGTH        (4)      /* Pressure + temperature data is 4 bytes long */
```
only use spaces for indentation